app/assets/stylesheets/collections/bulk-edit.scss

```diff
@@ -0,0 +1,51 @@
+.bulk-edit{
+  &-container {
```

Comment:
This classname is getting picked up by a CSS rule that says anything ending in "-container" gets `width: 70%;` and `margin: 0 auto;`. Not your fault for naming this class this way, but it's screwing with the sizing because that other CSS selector is too generic. 
@patrickwroach - Can you look into this? Search the project files for `[class*="-container"]` to see what I'm talking about.

Reply:
This is mildly intended. Any container class should follow similar rules, and then any specific changes should be built on top of them with the specific class name.  What changes need to be made to the bulk-edit-container?